### ISSUE TITLE
Nano: WindowsFeature resource does not work on Nano because Get-WmiObject command not found

### ISSUE THREAD

**Issue by NickMRamirez** [PowerShell/xPSDesiredStateConfiguration#257]
If I try to use the WindowsFeature resource, such as in the following, on a Windows 2016 Nano server:

```
Configuration Webserver 
{
    Import-DscResource -ModuleName xWebAdministration

    Node "172.28.128.9"
    {
        WindowsFeature "Webserver"
        {
            Name = "Web-Server"
            Ensure = "Present"
            IncludeAllSubFeature = $TRUE
        }
    }
}
```

On Nano, I get the following error:

```
VERBOSE: [nano]: LCM:  [ Start  Test     ]  [[WindowsFeature]Webserver]
The term 'Get-WmiObject' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
    + CategoryInfo          : ObjectNotFound: (Get-WmiObject:) [], CimException
    + FullyQualifiedErrorId : CommandNotFoundException
    + PSComputerName        : 172.28.128.9

You cannot call a method on a null-valued expression.
    + CategoryInfo          : InvalidOperation: (:) [], CimException
    + FullyQualifiedErrorId : InvokeMethodOnNull
    + PSComputerName        : 172.28.128.9

VERBOSE: [nano]: LCM:  [ End    Test     ]  [[WindowsFeature]Webserver]  in 0.2670 seconds.
```

On Nano, the Get-WmiObject command is not defined:

```
[127.0.0.1]: PS C:\Users\Administrator\Documents> Get-Command Get-WmiObject
Get-Command : The term 'Get-WmiObject' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
    + CategoryInfo          : ObjectNotFound: (Get-WmiObject:String) [Get-Command], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException,Microsoft.PowerShell.Commands.GetCommandCommand
```

**Comment by NickMRamirez** [comment]
We may be able to replace `Get-WmiObject` with `Get-CimInstance`, which is defined on Nano.

**Comment by kwirkykat** [comment]
@NickMRamirez Have you tested this fix on a Nano server?

**Comment by johlju** [comment]
Closing this as this seem to have been resolved in PR #258
